crates/ty_python_semantic/src/semantic_index/place.rs

```diff
@@ -775,7 +786,7 @@ impl std::fmt::Debug for PlaceTable {
 pub(super) struct PlaceTableBuilder {
     table: PlaceTable,
 
-    associated_place_ids: IndexVec<ScopedPlaceId, Vec<ScopedPlaceId>>,
+    associated_place_ids: IndexVec<ScopedPlaceId, SmallVec<[ScopedPlaceId; 4]>>,
```

Comment:
This `SmallVec` here is "free" as 4 `ScopedPlaceId`s + the tag fit into 24 bytes